src/pudl/transform/ferc1.py

```diff
@@ -431,11 +437,9 @@ def read_dbf_to_xbrl_map(dbf_table_name: str) -> pd.DataFrame:
             ],
         )
     # Select only the rows that pertain to dbf_table_name
-    row_map = row_map.loc[
-        row_map.sched_table_name == dbf_table_name,
-        ["report_year", "row_number", "row_type", "xbrl_factoid"],
-    ]
-    row_map = row_map.fillna(value={"dbf_only": False})
+    row_map = row_map.loc[row_map.sched_table_name == dbf_table_name].fillna(
+        value={"dbf_only": False}
+    )
```

Comment:
all i did here was remove the column selection which was removing `sched_table_name`. Since we are currently not using `dbf_only`, I'd propose removing this fillna altogether